app/controllers/buyer_controller.rb

```diff
@@ -7,6 +7,10 @@ def show
     @buyer = Buyer.find(params[:id])
   end
 
+  def update
```

Comment:
I think we have both a buyer controller as well as a buyers controller? we shouldnt have both